frontend/src/features/projects/spl/forms/CloseOutAdjustmentForm.tsx

```diff
@@ -50,7 +49,7 @@ const CloseOutAdjustment = (props: CloseOutAdjustmentProps) => {
       </Form.Row>
       <ProjectNotes
         label="Adjustment to Prior Year Sale Notes"
-        field={`notes[${NoteTypes.Adjustment}].note`}
```

Comment:
Using array positioning is an invalid implementation and will result in data corruption